src/record/record.js

```diff
@@ -249,17 +250,17 @@ Record.prototype.whenReady = function( callback ) {
  */
 Record.prototype._$onMessage = function( message ) {
 	if( message.action === C.ACTIONS.READ ) {
-		this._clearTimeouts();
+		clearTimeout( this._readTimeout );
```

Comment:
Yup, but only getting a read without preceding ACK should be a valid case. Otherwise we'll get the data, the record is ready and then the ACK timeout triggers and throws an error